lib/runtime/addon.js

```diff
@@ -157,8 +157,7 @@ export default class Addon {
     forIn(this.pkg.dependencies, (version, pkgName) => {
       let pkgMainPath = resolve.sync(pkgName, { basedir: this.dir });
       let pkgJSONPath = path.resolve(findup('package.json', { cwd: pkgMainPath }));
-      // TODO this shouldn't need to be aware of dist
-      let pkgDir = path.join(path.dirname(pkgJSONPath), 'dist');
+      let pkgDir = path.dirname(pkgJSONPath);
```

Comment:
@acorncom yea, we can basically either keep the `dist` here and remove it from the addon package.json blueprint, or vice versa.

I think removing it from here makes more sense, since that leaves the package.json "main" as a valid path.